src/views/Receiver.vue

```diff
@@ -249,6 +257,9 @@ export default {
         return null;
       }
     },
+    getVolume() {
```

Comment:
A computed variable shouldn't have `get` in the name (stylistically); this is a bit annoying, since the `data` variable is also called `volume`, so we'll have to rename the data parameter, *or, even better,* maybe we could call this `volumeForSlider` since it's `* 100` specifically for the slider widget.